modules/telestion-services/build.gradle

```diff
@@ -46,7 +46,7 @@ dependencies {
     implementation project(':modules:telestion-api')
 
     implementation group: 'com.fazecast', name: 'jSerialComm', version: '2.7.0'
-    implementation 'com.google.guava:guava:30.1.1-jre'
+//    implementation 'com.google.guava:guava:30.1.1-jre'
```

Comment:
Why comment out? Please remove the line.

Reply:
Will be done.